packages/bibtex/csl/engine.lua

```diff
@@ -424,7 +436,9 @@ function CslEngine:_layout (options, content, entries)
    if self.mode == "citation" then
       for _, entry in ipairs(entries) do
          self:_prerender()
-         local elem = self:_render_children(content, entry)
+         local elem = self:_render_children(content, entry, {
+            delimiter = options.delimiter
```

Comment:
Wrong! There delimiter on the citation layout applies between entries (taken care of just below) and shouldn't be passed as context for an individual entry (=> breaks e.g. with acta-philosophica that starts with an immediate `<choose>`)

Reply:
Fixed